.travis.yml

```diff
@@ -21,13 +21,14 @@ before_script:
   - if [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo apt-get --assume-yes --force-yes install cmake libeigen3-dev icub-common libv4lconvert0 libv4l-dev; fi
 
   # macOS dependencies
-  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew tap robotology/cask; fi
-  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew install --only-dependencies yarp; fi
-  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew install gsl goocanvas sdl sdl_gfx sdl_image homebrew/science/ipopt; fi
+  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew update; fi
   # Fix for opencv and numpy already installed by system
   - if [ "$TRAVIS_OS_NAME" = osx ]; then rm '/usr/local/bin/f2py'; fi
```

Comment:
To agree with the file convention on strings, I would change `osx` into `"osx"` in this and the following line.